docs/tool-prerequisites.md

```diff
@@ -47,3 +47,10 @@ mpdev /scripts/doctor.py
 ```
 
 Save your gcloud project to an environment variable.
+
+
+## Install Application CRD
+
+```shell
+kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
```

Comment:
Can you update [this line](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/89bdb09b875bde5a55f1c8cf31592b7232de99b7/scripts/doctor.py#L202) to remove `curl` too?

Reply:
Done!